Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -965,7 +873,8 @@ static u32 TranslateBlockAddress(const u32 addr, const XCheckTLBFlag _Flag)
 }
 
 // Translate effective address using BAT or PAT.  Returns 0 if the address cannot be translated.
-u32 TranslateAddress(const u32 _Address, const XCheckTLBFlag _Flag)
+template <const XCheckTLBFlag _Flag>
```

Comment:
That's probably the most straightforward thing to do.  I mean, it's a little redundant, but it serves as a hint to help avoid obscure linker errors if the code is refactored.

Alternatively, you could make TranslateAddress static and add a separate entry point for PPCDebugInterface.cpp .